config/quality.gradle

```diff
@@ -21,8 +21,6 @@ def reportsDir = "${project.buildDir}/reports"
 
 apply plugin: 'checkstyle'
 
-checkstyle.toolVersion = '7.6.1'
```

Comment:
Removing this line updates the checkstyle version to the latest?
How did you verify that it indeed used the latest version?

Reply:
as per @huangyz0918  suggestion, i had it removed and let the android studio sync the gradle

Reply:
in my #200 , i had it updated to 8.18

Reply:
I have to clarify, @iamvasuh I mean, the default version of checkstyle depends on your gradle version, it's not `the` latest among all the checkstyle versions. it will update automatically when your gradle updates.

Reply:
You can refer to my [comment](https://github.com/opendatakit/skunkworks-crow/pull/200#issuecomment-474251455),

>for example, from Gradle 3.3 onwards, the default checkstyle version is 6.19; before, it was 5.9. Only Gradle versions prior to 2.4 used even older versions of checkstyle.

> So, as we mentioned above, we can use the default checkstyle version, it will auto update as our Gradle version updates, or, we can just update the checkstyle version to 8.18 like your commit.

Reply:
i checked my gradle version, its 4.4 right now , so after making changes to the checkstyle plugin version(i.e. removing the version no. , excluding the whitespace modules from the checkstyle.xml file and letting the studio do its gradle sync ) it's verified that the project is using the latest version of checktyle plugin. @shobhitagarwal1612 @huangyz0918 
![Screen Shot 2019-03-22 at 10 35 36 AM](https://user-images.githubusercontent.com/42612183/54801842-bd822200-4c8e-11e9-99d9-4957df3caa77.png)